### ISSUE TITLE
Karma and Todos for Yosef Raisman

### ISSUE THREAD

**Issue by zvili** [TechnionYP5777/Bugquery#4]
_From @yosefraisman on October 30, 2016 11:23_

- [x]  Use a picture, photo or distinct avatar. Don''t use an automatic icon
- [x]  Make sure you present yourself by a readable nick
- [x]  Describe yourself in the bio page
- [x]  State that you are (not) in search for a job

_Copied from original issue: SpartanRefactoring/Spartanizer#616_

**Comment by zvili** [comment]
_From @yosefraisman on October 30, 2016 11:24_

@YossiGil: Done.

**Comment by zvili** [comment]
_From @SharonKL on October 30, 2016 12:20_

One of the best karma pages I have ever seen, well done!

**Comment by zvili** [comment]
_From @AviadCo on October 30, 2016 18:40_

Great karma page bro ;)

**Comment by zvili** [comment]
_From @yonzarecki on October 30, 2016 19:36_

I knew the best karma page would come last !

**Comment by zvili** [comment]
_From @ZivIzhar on October 30, 2016 22:4_

meow meow mitsi meow? karmeow

**Comment by zvili** [comment]
_From @RoeiRaz on October 31, 2016 7:11_

Solid karma page, well done 👍

**Comment by zvili** [comment]
_From @Shahar-Y on October 31, 2016 8:32_

great karma page! visit mine at: #665 , I like ABBA

**Comment by zvili** [comment]
_From @roy-shchory on October 31, 2016 14:8_

im contacting you in regards of your awsome band! we would like to sign you! this is totaly not a scam!
for the details go to #621

**Comment by yossigil** [comment]
Can you summarize here all issues you dealt with towards Sprint 1?

@AdiOmari

**Comment by yosefraisman** [comment]
Hi @yossigil, my contribution so far:
1. Implementation of Plug-in
2. Preliminary Plug-in Research
3. Meeting Protocols, Documentation (Wiki)

**Comment by yossigil** [comment]
I asked for issues you completed.  @zvili

**Comment by yosefraisman** [comment]
Sorry, I misunderstood. Issue #7 is the only one I dealt with that's closed. Plug-in implementation issues are #12 and #13, and we don't have an issue for protocols and Wiki.

**Comment by yossigil** [comment]
So, you should probably create issues for these...

**Comment by yosefraisman** [comment]
Created issue #18 for weekly protocols (not sure what milestone I should set). Also #19 for our README file, which I take responsibility for.

**Comment by yossigil** [comment]
Meeting protocols is an issue which has no end. No one can know when it would finish.
